Assets/Scripts/SceneManager.cs

```diff
@@ -0,0 +1,8 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SceneManager : MonoBehaviour
+{
+    public int nextSceneNum;
+}
```

Comment:
I would take all of your scene related code and move it here rather than the game manager.

Reply:
ok so will have have to add the on trigger enter on the door rather then the interact script on PLayer?